published/007-2016-March-13.md

```diff
@@ -0,0 +1,58 @@
+# IPFS Weekly #7
+
+[IPFS](//ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open. In these posts, we highlight some of the development that has happened in the past week. For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [GitHub](//github.com/ipfs) or join us on [IRC](//webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+If you would like to get this update as an email, sign up for our [weekly newsletter](//tinyletter.com/ipfsweekly)!
+
+Here are some of the highlights for the [second week of March](//github.com/ipfs/pm/issues/97):
+
+## Updates
+
+### [__Repo__](//github.com/)
+
+### Orbit
+
+[Orbit](https://github.com/haadcode/orbit) is back in active development and going through a major code base overhaul. It now uses orbit-db as it's database layer.
+
+[orbit-db](https://github.com/haadcode/orbit-db) is a KV-store and Event Log on top of IPFS which allows developers to use IPFS as a database. Last week orbit-db got big performance and stability improvements and is now using CRDTs for eventual consistency.
+
+### [js-multiaddr](https://github.com/jbenet/js-multiaddr)
+
+_@diasdavid:_ I don't know what these mean. Why are these changes important? Can you explain them in English a bit more? Also, these weren't mentioned in the sprint hangout as things done last week - why not?
```

Comment:
[> multiaddr JS impl was upgraded, now it matches go-ipfs features which means protocols whose code's required a varint https://github.com/jbenet/js-multiaddr](https://github.com/ipfs/weekly/issues/27#issuecomment-196476291)

What this means is that js-multiaddr now supports protocols that need to be encoded with varints. A varint is a integer that uses only the number of bytes it needs to be described by using a continuation bit (notes: http://web.stanford.edu/class/cs276/Jeff-Dean-compression-slides.pdf). Before this addition, we were not able to declare http, websockets or even ipfs multiaddrs because these protocol have a code that doesn't fit in a byte (max: 2^8 = 256). You can check the encoding table at https://github.com/jbenet/js-multiaddr/blob/master/src/protocols.csv.

Hope this helps, ping me if there are still questions :)

note: github doesn't send notifications by mentions on files, you have to explicitly mention me in a issue or PR, like @jbenet did.